datajunction-ui/src/app/components/Search.jsx

```diff
@@ -0,0 +1,82 @@
+import { useState, useEffect, useContext } from 'react';
+import DJClientContext from '../providers/djclient';
+import Fuse from 'fuse.js';
+
+import './search.css';
+
+export default function Search() {
+  const [fuse, setFuse] = useState();
+  const [searchValue, setSearchValue] = useState('');
+  const [searchResults, setSearchResults] = useState([]);
+
+  const djClient = useContext(DJClientContext).DataJunctionAPI;
+
+  const truncate = str => {
+    return str.length > 100 ? str.substring(0, 90) + '...' : str;
+  };
+  useEffect(() => {
+    const fetchNodes = async () => {
+      const data = await djClient.nodeDetails();
+      const fuse = new Fuse(data, {
+        keys: [
```

Comment:
Wanted to point out this fuse index configuration which would be useful to adjust as we see the behavior in the field. They apply weights to the indexed objects.

### Weights
Name = 1
Description = 2
Display name = 3
Type = 4

For example, if something like "source", "transform", "metric", "dimension", or "cube" exists in the search, that would almost certainly bubble those node types up to the top since the index score for the type field is weighted at 4x.